src/status_im2/contexts/onboarding/common/syncing/view.cljs

```diff
@@ -0,0 +1,50 @@
+(ns status-im2.contexts.onboarding.common.syncing.view
+  [:require
+   [quo2.core :as quo]
+   [utils.i18n :as i18n]
+   [quo2.foundations.colors :as colors]
+   [react-native.core :as rn]
+   [status-im2.common.not-implemented :as not-implemented]
+   [status-im2.contexts.onboarding.common.syncing.style :as style]])
+
+;; TODO replace with settings component
```

Comment:
please create an issue for this TODO :)

Reply:
hmm, I think I already have a similar page to this implemented (it looks like just the background is different)
https://github.com/status-im/status-mobile/blob/develop/src/status_im2/contexts/syncing/view.cljs#L54

Let's try to re-use the code for both pages if so. Ping me and we can pair on it :)

Reply:
I realized its actually a component but to use your code I moved render-device function from the ns quoted above to a common ns but still needs an implementation
https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?node-id=1650%3A31097&t=0fFyi5RetW3JBgFI-1